lua/catppuccin/init.lua

```diff
@@ -119,11 +119,14 @@ function M.setup(user_conf)
 		file:close()
 	end
 
-	local stat = vim.loop.fs_stat(debug.getinfo(2).source:sub(2)) -- Get user config stat
-	local git = vim.loop.fs_stat(debug.getinfo(1).source:sub(2, -24) .. ".git" .. M.path_sep .. "ORIG_HEAD") -- Get git stat
-	local cur_date = (stat and stat.mtime.sec or 0) + (git and git.mtime.sec or 0)
+	local config_path = debug.getinfo(2).source:sub(2) -- Get user config path
+	local git_path = debug.getinfo(1).source:sub(2, -24) .. ".git" .. M.path_sep .. "ORIG_HEAD" -- Get git path
+	local config = vim.fn.getftime(config_path) -- getftime is 2 time faster than fs_stat, for benchmark see #352
+	local git = vim.fn.getftime(git_path) -- Parsed config & git stat
```

Comment:
`git` is `-1` here because `git_path` doesn't exists. Remember?

Reply:
@stasjok Ah yeah, will fix it